iocage_lib/ioc_start.py

```diff
@@ -66,7 +68,11 @@ def __init__(self, uuid, path, silent=False, callback=None,
                                                    silent=True).json_set_value
 
             self.exec_fib = self.conf["exec_fib"]
-            self.__start_jail__()
+            try:
+                self.__start_jail__()
+            except SystemExit as e:
```

Comment:
That does mean that if iocage is ran noninteractively (freenas mw) then the exceptions will never be suppressed. Is that the desired behavior?